ex2/part 2 - sport game.py

```diff
@@ -0,0 +1,68 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+def del_space1(text: str):
```

Comment:
This method is complicated and very hard to read.
Let's simplify it ! Hooray !

Follow the following steps * in order*:
1. Write in Hebrew what it does.
2. Rename the function to a more verbose name.
3. Write one line documentation in English.
4. Write tests for the function. The tests should cover all interesting cases and edge cases.
5. Then and only then (after we know what it should do AND we have tests in place) -
rewrite the function.
Use the following methods of string -
`replace`
`strip`
I reckon that one can implement the required functionality with only two simple lines.